docs/Regenerating Server and Client.md

```diff
@@ -1,115 +1,115 @@
 # Regenerating Servers and Clients
 
-Below are instructions for regenerating the grpc-labview code for MeasurementLink servers and clients.
+Below are instructions for regenerating the grpc-labview code for measurement plug-in servers and clients.
 
 ## Setup
 
 - Install LabVIEW 2020 SP1
 - Install all `grpc-labview` VI packages from the desired `grpc-labview` [release](https://github.com/ni/grpc-labview/releases)
-- Clone the `measurementlink-labview` github repo
+- Clone the `measurement-plugin-labview` github repo
 
-## MeasurementLink Discovery Client
+## Discovery Client
 
 The following are instructions for regenerating the Discovery Client.
 
-- Open `\Source\Runtime\Discovery Client.lvproj`
-- Delete the entire `MeasurementLink Discovery Client` virtual folder and save the project
-- Delete the entire `MeasurementLink Discovery Client` on disc
+- Open `\Source\gRPC\Generated APIs\ni\measurementlink\discovery\v1\ni.measurementlink.discovery.v1.api.lvproj`
+- Delete the entire `ni.measurementlink.discovery.v1` virtual folder and save the project
```

Comment:
I made all the updates in this file based on recent diffs. I did not try to follow the steps myself.

Note that this particular virtual folder does end in ".api" like the others do, because the .lvproj file doesn't include ".api" in the Folder name in this case.